htsget-http-core/src/lib.rs

```diff
@@ -132,7 +133,7 @@ mod tests {
       Ok(JsonResponse::from_response(Response::new(
         Format::Bam,
         vec![Url::new(format!(
-          "file://{}",
+          "https://127.0.0.1:8081{}",
```

Comment:
I guess those hardcoded ip/port is the default just for testing?

Reply:
Yeah I just added those for testing. Potentially we could introduce some constants at the top of test modules which refactor hardcoded values, as there is a bit of repetition. But I'm not sure if it's is worth it?

Reply:
If those are just for the testing bits, nevermind 👍🏻